torchsynth/synth.py

```diff
@@ -216,21 +216,26 @@ def randomize(self, seed: Optional[int] = None):
         Randomize all parameters
         """
         if seed is not None:
-            cpu_rng = torch.Generator(device="cpu").manual_seed(seed)
-            for parameter in self.parameters():
+            # Generate batch_size x parameter number of random values
+            # Reseed the random number generator for every item in the batch
+            cpu_rng = torch.Generator(device="cpu")
+            parameters = [param for param in self.parameters()]
```

Comment:
Does this always return in a deterministic order? I feel like sometimes I've seen it be randomized?